src/js/components/TabPanel/TabView.js

```diff
@@ -88,7 +88,7 @@ export default class TabView extends Component {
 
           </div>
           <h3 className='tab-view__mobile-header mobile-show'>{text[language].DATA}</h3>
-          <InfoWindow map={map} />
+          <InfoWindow map={map} {...this.props} />
```

Comment:
@KaylaKremer Let's **just** pass the prop `editingEnabled` if that' all we need!